packages/agreement-process/src/services/agreementService.ts

```diff
@@ -416,7 +416,13 @@ async function createAndCopyDocumentsForClonedAgreement(
         ),
       };
     })
-  );
+  ).catch((error) => {
+    logger.error(
+      `Error copying documents' files for agreement ${clonedAgreement.id} : ${error}`
+    );
+    throw error;
+  });
```

Comment:
ok from my side to split the activities, as long as the 283 is done right after the current PR, because in my opinion the different error handling is part of this feature.
The sooner we do the refactor, the lesser code will use the old approach and will need to be refactored